README.md

```diff
@@ -14,17 +14,38 @@ Repository for TDR transfer code
 
 * In the client settings, change the login theme to govuk.
 
+* Set "Root URL" to `http://localhost:9000`
+
+* Set "Valid redirect URIs" to `http://localhost:9000/*`
+
 * Generate a [secret](https://www.keycloak.org/docs/latest/server_admin/#_client-credentials)
 
-* Set the secret in auth.secret in [application.conf](conf/application.conf).
+* In the [application.conf](conf/application.conf) file find the auth.secret key and set its value to the secret you have just generated
+
+### User accounts
 
 * Create a new [user](https://www.keycloak.org/docs/latest/getting_started/index.html#_create-new-user) in the tdr realm.
 
 * Start redis locally.
 
 `docker run -d --name redis -p 6379:6379 redis`
 
+### Static assets
+
+* If npm is not installed install [nvm](https://github.com/nvm-sh/nvm) in root directory.
+
+* Once nvm is installed:
+    `nvm install 13.6`
+
+* `cd` into tdr-transfer-frontend in terminal
+
+* run  `npm install` again, then `npm run build`
```

Comment:
We can drop the "again", because this is now the first time `npm install` is mentioned.

Reply:
Ah, I'll change that! 👌